Tasks/GithubRelease/main.ts

```diff
@@ -0,0 +1,30 @@
+import tl = require("vsts-task-lib/task");
+import path = require("path");
+import { createReleaseAction, editReleaseAction, discardReleaseAction } from "./operations/Action";
+
+async function run(): Promise<void> {
+
+    try {
+        var taskManifestPath = path.join(__dirname, "task.json");
+        tl.debug("Setting resource path to " + taskManifestPath);
+        tl.setResourcePath(taskManifestPath);        
+
+        let action = tl.getInput("action");
+        tl.debug("Github action = " + action);
```

Comment:
I think tl.getInput itself prints the input value. Can you check?

Reply:
yes